CHANGELOG.md

```diff
@@ -36,6 +36,7 @@
 - **General:** Introduce new GCP Storage Scaler ([#2628](https://github.com/kedacore/keda/issues/2628))
 - **General:** Introduce ARM-based container image for KEDA ([#2263](https://github.com/kedacore/keda/issues/2263)|[#2262](https://github.com/kedacore/keda/issues/2262))
 - **General:** Provide support for authentication via Azure Key Vault ([#900](https://github.com/kedacore/keda/issues/900))
+- **General:** Provide support for non-public clouds for Azure Key Vault ([#2733](https://github.com/kedacore/keda/issues/2733))
```

Comment:
Just a nit: I'd merge this with the previous entry and just add the second link to the issue 2733 (the same as it is done on line 37).

Reply:
Done.